src/gpu/GPUFieldDesc.hpp

```diff
@@ -17,7 +17,8 @@
 // Used to send e.g. formatting request to GPU
 struct GPUFieldDesc
 {
-	const char* data;
+	const char* readDataPtr;
+	char* writeDataPtr;
```

Comment:
I'd appreciate an union here with an appropriate comment.

Reply:
A union turns out to be unsafe. I will provide an appropriate comment only.